.github/workflows/lint-test.yaml

```diff
@@ -37,6 +37,62 @@ jobs:
       - run: cd ./ecosystem/indexer-server/typescript && yarn install
       - run: cd ./ecosystem/indexer-server/typescript && yarn lint
 
+  webplatform-lint:
```

Comment:
can we call this community-platform-lint or something like that?

Reply:
Done.